apps/web/src/components/ClickableLogo/ClickableLogo.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { screen, render, fireEvent } from '@testing-library/react';
```

Comment:
Can you refactor to use `useEvent`?

<img width="1042" alt="Screenshot 2023-03-24 at 7 48 22 AM" src="https://user-images.githubusercontent.com/12022922/227558459-38fc6ee3-62fe-4bed-b77f-ca3bf7e5a72c.png">

https://testing-library.com/docs/user-event/intro